docs/mongo-setup.md

```diff
@@ -7,11 +7,20 @@ add yourself as an admin user.
 
    ![collections](./images/collections.png)
 
-2. Click "+ Create Database" to create a database. Make sure you name it "database" and create a collection called "users".
+2 If you see "+ Create Database", click that to create a database.
```

Comment:
I think this is missing the period in the 2. to make this a list

Reply:
Good catch.  Indentation was incorrect too.    Fixed now, I think.